main.go

```
@@ -48,12 +50,30 @@ func registerV1Alpha(rwdb, rodb *sqlx.DB, statements *graphstore.Statements, ser
 	services.RegisterTopologyService(server, graphStoreClient)
 }
 
+// Maps the available drivers to a DBMS name
+func getDBMSName(driverName string) string {
+	switch driverName {
+	case "mysql", "go-mysql", "mymysql":
+		return graphstore.MySQL
+
+	case "pgx", "pq", "gopgsqldriver":
+		return graphstore.Postgres
+
+	case "go-sqlite3", "gosqlite", "go-sqlite", "sqlite":
+		return graphstore.Sqlite
+
+	default:
+		return graphstore.Unknown
```

I think we want this function to return a `(string, error)`. If the driver is unrecognized, it should return an error instead of an unknown value.

---

Yeah that sounds better, so we could panic.